src/gazebo_vision_plugin.cpp

```diff
@@ -88,6 +88,22 @@ void VisionPlugin::getSdfParams(sdf::ElementPtr sdf)
     _noise_density = kDefaultNoiseDensity;
     gzwarn << "[gazebo_vision_plugin] Using default noise density of " << _noise_density << " (m) / sqrt(hz)\n";
   }
+
+  if(sdf->HasElement("enableRosOdom")) {
```

Comment:
Please look at https://github.com/EliaTarasov/sitl_gazebo/blob/7c0526d73d4d4dcba9d1baaa4c7b4c920b496105/src/gazebo_vision_plugin.cpp#L132
It is used in case when plugins are built with ROS but this particular plugin is need to be used without ROS.